be/src/exec/vectorized/sorting/merge_column.cpp

```diff
@@ -435,16 +472,16 @@ Status merge_sorted_chunks_two_way(const SortDescs& sort_desc, const std::vector
 }
 
 Status merge_sorted_chunks(const SortDescs& descs, const std::vector<ExprContext*>* sort_exprs,
-                           const std::vector<ChunkPtr>& chunks, SortedRuns* output, size_t limit) {
-    if (chunks.empty()) {
-        return Status::OK();
-    }
+                           const std::vector<SortedRuns>& chunks, SortedRuns* output, size_t limit) {
     std::deque<SortedRuns> queue;
     for (auto& chunk : chunks) {
-        if (!chunk->is_empty()) {
-            queue.push_back(SortedRun(chunk, sort_exprs));
+        if (chunk.num_chunks() > 0) {
```

Comment:
what's difference of `!chunk->is_empty()` and `chunk.num_chunks()>0`

Reply:
which is SortedRuns, SortedRuns has num_chunks method, no is_empty method. maybe need to rename.

Reply:
renamed